Doc/library/ssl.rst

```diff
@@ -764,6 +769,15 @@ Constants
 
    .. versionadded:: 3.4
 
+.. data:: OP_NO_TLSv1_3
+
+   Prevents a TLSv1.3 connection. This option is only applicable in conjunction
+   with :const:`PROTOCOL_TLS`. It prevents the peers from choosing TLSv1.3 as
+   the protocol version. TLS 1.3 is available with OpenSSL 1.1.1 or later.
+   With older versions, the flag defaults to *0*.
```

Comment:
We don't currently expose `OP_NO_TLSv1_2` if your OpenSSL is old enough to not have TLS1.2. Why is this flag special? Should we just change all of them for consistency?